hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -2321,6 +2324,9 @@ protected String reconfigurePropertyImpl(String property, String newVal)
       return reconfigureSlowNodesParameters(datanodeManager, property, newVal);
     } else if (property.equals(DFS_BLOCK_INVALIDATE_LIMIT_KEY)) {
       return reconfigureBlockInvalidateLimit(datanodeManager, property, newVal);
+    } else if (property.equals(DFS_NAMENODE_DECOMMISSION_BACKOFF_MONITOR_PENDING_LIMIT)) {
```

Comment:
Hi @haiyang1987 , can we also make `DFS_NAMENODE_DECOMMISSION_BACKOFF_MONITOR_PENDING_BLOCKS_PER_LOCK_DEFAULT` reconfigurable? And create a method for `DecommissionBackoffMonitor`. Just like `reconfigureSlowNodesParameters()`.

Reply:
Hi @tomscut, thanks for your comment.
yeah,  can create new method reconfigureDecommissionBackoffMonitorParameters() to update DecommissionBackoffMonitor related parameters.

HDFS-16810 to support ‘DFS_NAMENODE_DECOMMISSION_BACKOFF_MONITOR_PENDING_BLOCKS_PER_LOCK_DEFAULT’ reconfigurable

Do we need to merge 2 issues into one to support?

Reply:
> Hi @tomscut, thanks for your comment. yeah, can create new method reconfigureDecommissionBackoffMonitorParameters() to update DecommissionBackoffMonitor related parameters.
> 
> [HDFS-16810](https://issues.apache.org/jira/browse/HDFS-16810) to support ‘DFS_NAMENODE_DECOMMISSION_BACKOFF_MONITOR_PENDING_BLOCKS_PER_LOCK_DEFAULT’ reconfigurable
> 
> Do we need to merge 2 issues into one to support?

I think it could be merged together to make it clearer. Thanks.

Reply:
@tomscut Thanks for comment, 
I will  merge this two issues support into here, update PR later.